x-pack/plugins/cases/public/components/all_cases/nav_buttons.tsx

```diff
@@ -54,7 +54,7 @@ export const NavButtons: FunctionComponent<Props> = ({ actionsErrors }) => {
           <EuiFlexItem grow={false}>
             <ConfigureCaseButton
               label={i18n.CONFIGURE_CASES_BUTTON}
-              isDisabled={!isEmpty(actionsErrors)}
+              isDisabled={false}
```

Comment:
Maybe we can just get rid of it? 😄 `ConfigureCaseButton` doesn't seem to be used anywhere else.